code/go/0chain.net/miner/protocol_block_main.go

```diff
@@ -307,3 +307,8 @@ func (mc *Chain) GenerateBlock(ctx context.Context, b *block.Block,
 	node.Self.Underlying().Info.AvgBlockTxns = int(math.Round(bsHistogram.Mean()))
 	return nil
 }
+
+/*UpdateFinalizedBlock - update the latest finalized block */
```

Comment:
why do we need this func?

Reply:
The main implementation (`updateFinalizedBlock`) stays at the base code part shared to all builds. Wrapping functions like that help ignore useless code copying.

Reply:
The main implementation (`updateFinalizedBlock`) stays at the base code part shared to all builds. Wrapping functions like that help ignore useless code copying.